doc/query.md

```diff
@@ -72,9 +105,47 @@ if (name == null) {
 }
 ```
 
-`ensureIndexed(List<Object> fieldNames, String indexName)` returns the name of the index if it is successful, otherwise it returns `null`.
+Since text search relies on SQLite FTS, we must ensure that SQLite was compiled with both the `ENABLE_FTS3` and `ENABLE_FTS3_PARENTHESIS` options.  To verify that text search is enabled and that a text index can be created use `isTextSearchEnabled()` before attempting to create a text index.  If text search is not enabled see [compiling and enabling SQLite FTS][enableFTS] and [SQLite Android Bindings][androidBind] for details. 
```

Comment:
Added the `Indexing for text search` sub-heading here.  Done in ee198020c5b7e28d8fcf044f68f77f22fa1db7e2.